contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -371,8 +375,8 @@ func (s *SchedulerServer) prepareExecutorInfo(hks hyperkube.Interface) (*mesos.E
 		log.Infof("Detected %d staticPods in Configuration.", numberStaticPods)
 
 		info.Resources = []*mesos.Resource{
-			mutil.NewScalarResource("cpus", float64(numberStaticPods)*podtask.DefaultContainerCpus),
-			mutil.NewScalarResource("mem", float64(numberStaticPods)*podtask.DefaultContainerMem),
+			mutil.NewScalarResource("cpus", float64(numberStaticPods)*float64(podtask.DefaultContainerCpus)),
```

Comment:
we should do one of two things here:
(a) properly calculate the resource requirements of these static pods
(b) file an issue to accomplish (a); and leave a TODO(sttts) here